### DISCUSSION TITLE
visual studio

### DISCUSSION THREAD

**Discussion by malihasaeed1** [community/community#36460] [Copilot Conversations] [answered]
after purchase of github co pilot how do you use it in visual studios

**Comment by davecheney** [comment, marked as answer]
@malihasaeed1 welcome to Copilot, please consult the Visual Studio getting started document https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio

**Comment by ossanna16** [comment]
Hey there @malihasaeed1 and thank you for asking a good question! I'm glad you found our community 🙂
